examples/tracing/jaeger/pyroscope.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 cors:
-  allowed-origins: ['http://localhost:3000', 'http://localhost:16686']
+  allowed-origins:
+    ['http://localhost:3000', 'http://localhost:16686', 'http://localhost:4000']
```

Comment:
@pavelpashkovsky jaeger-ui is served on port :4000 which is why we continued to get cors error